wasm_tests/i64.wast

```diff
@@ -0,0 +1,494 @@
+(; (assert_return (invoke "gt_u" (i64.const 0x8000000000000000) (i64.const 0x7fffffffffffffff)) (i32.const 1)) ;)
+(; (assert_return (invoke "gt_u" (i64.const 0x7fffffffffffffff) (i64.const 0x8000000000000000)) (i32.const 0)) ;)
+(;;)
+(; (assert_return (invoke "ge_s" (i64.const 0) (i64.const 0)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 1) (i64.const 1)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const -1) (i64.const 1)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x8000000000000000) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x7fffffffffffffff) (i64.const 0x7fffffffffffffff)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const -1) (i64.const -1)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 1) (i64.const 0)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0) (i64.const 1)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x8000000000000000) (i64.const 0)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x8000000000000000) (i64.const -1)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_s" (i64.const -1) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x8000000000000000) (i64.const 0x7fffffffffffffff)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_s" (i64.const 0x7fffffffffffffff) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(;;)
+(; (assert_return (invoke "ge_u" (i64.const 0) (i64.const 0)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 1) (i64.const 1)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const -1) (i64.const 1)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x8000000000000000) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x7fffffffffffffff) (i64.const 0x7fffffffffffffff)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const -1) (i64.const -1)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 1) (i64.const 0)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0) (i64.const 1)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x8000000000000000) (i64.const 0)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0) (i64.const 0x8000000000000000)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x8000000000000000) (i64.const -1)) (i32.const 0)) ;)
+(; (assert_return (invoke "ge_u" (i64.const -1) (i64.const 0x8000000000000000)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x8000000000000000) (i64.const 0x7fffffffffffffff)) (i32.const 1)) ;)
+(; (assert_return (invoke "ge_u" (i64.const 0x7fffffffffffffff) (i64.const 0x8000000000000000)) (i32.const 0)) ;)
+
+
+;; Type check
```

Comment:
I didn't add the equivalent type check assertions to the i32.wast tests. Should they be there also?

Reply:
can remove, they're fully ignored rn